### ISSUE TITLE
[EN][Quest][Icecrown] The Sum Is Greater the Parts

### ISSUE THREAD

**Issue by Gandalior** [Atlantiss/BugTracker#8545]
[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

[//]: # (On the Bugtracker all titles of reports must be written in English so that everyone, no matter which language they use everyday, could search and check if the problem already has been reported by another player. When describing the problem you can use English as well as Polish and in nearest future - others. It is important to determine at the beginning of a title in which language it is written by using symbols [EN] or [PL].)

[//]: # (Enclose links to things related to the bug using http://wowhead.com or http://cata.openwow.com)
[//]: # (Write your tickets according to the format:)
[//]: # ([EN][Quest][Zone] A Vision of the Past)
[//]: # ([EN][NPC] Thoralius the Wise)
[//]: # ([EN][Spell][Class] Frostfire Bolt)
[//]: # ([PL][Talent][Class] Brain Freeze)
[//]: # ([EN][Glyph][Class] Glyph of Frostfire Bolt)
[//]: # ([EN][Npc][Drop] Thoralius the Wise)
[//]: # ([EN][Web] Armory doesnt work)

**Description**: You are supposed to mount "Nergeld" ( http://www.wowhead.com/npc=30403/nergeld ) but it's not possible to do, he appears as a friendly NPC but can't be mounted

**Current behaviour**: Can't mount Nergeld to begin killing adds to complete quest.

**Expected behaviour**: You Should be able to mount Nergeld, then adds start spawning, when you have killed enough adds, "Dr. Terrible" ( http://www.wowhead.com/npc=30404/dr-terrible ) should spawn and be killable to complete the quest.

**Steps to reproduce the problem**:

1. Pick up the Book ( http://www.wowhead.com/item=42772/dr-terribles-building-a-better-flesh-giant ) that starts the quest from the "Apprentice Osterkilgr" ( http://www.wowhead.com/npc=30409/apprentice-osterkilgr ) 
2.  Walk to the other room and Nergeld is there but can't be mounted
3.  Quest can't be completed

**Revision**: Current server version

**Comment by Piklo** [comment]
you can see current server revision by typing ".s in" in chat
example:
![revision](https://cloud.githubusercontent.com/assets/16057987/20240620/5b56a446-a91d-11e6-9b05-532d6f03ae00.PNG)

**Comment by 1Andas** [comment]
Confirmed. This problem still persists (Nergeld still can't be mounted).

**Comment by podatekvat** [comment]
rev. 5418 - it's still not scripted properly. Would be nice to be able to complete it couse this quest blocks more quests in the chain quest and in addition loremaster of Icecrown achievement also.

**Comment by Samineh** [comment]
Bump :)

**Comment by poleloa** [comment]
I agree with you PodatekVat ,if mean scripting . That's why i haven't completed achivement Icecrown :
The Final Goal (124/140)yet.
